setup.cfg

```diff
@@ -1,73 +0,0 @@
-[metadata]
-name = asv
-description = Airspeed Velocity: A simple Python history benchmarking tool
-long_description = file: README.rst, CHANGES.rst
-author = Michael Droettboom
-author_email = mdroe@stsci.edu
-license = BSD-3-Clause
-license_file = LICENSE.rst
-platforms = any
-classifiers =
-    Development Status :: 5 - Production/Stable
-    Environment :: Console
-    Environment :: Web Environment
-    License :: OSI Approved :: BSD License
-    Operating System :: OS Independent
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Topic :: Software Development :: Testing
-project_urls =
-    Source Code = https://github.com/airspeed-velocity/asv
-    Documentation = https://asv.readthedocs.io/en/stable/
-
-[options]
-zip_safe = False
-include_package_data = True
-packages = find:
-python_requires = >=3.7
-
-[options.extras_require]
-test =
-    pytest
-    pytest-xdist
-    pytest-timeout
-    pytest-rerunfailures>=10.0
-    filelock
-    virtualenv
-    numpy
-    scipy; platform_python_implementation != "PyPy"
-    feedparser
-    selenium
-    python-hglib
-doc =
-    sphinx
-    sphinx_bootstrap_theme
-dev =
-    flake8
-    isort
-hg =
-    python-hglib
-
-[options.entry_points]
-console_scripts =
-    asv = asv.__main__:main
```

Comment:
How is this represented in the pyproject.toml?

Reply:
`entry_points.console_scripts` is represented by `project.scripts` in `pyproject.toml` (which is a bit confusing, as there was already a `scripts=` parameter which was not included in PEP621)
https://github.com/airspeed-velocity/asv/blob/c9cdb534eed02be1dde78630b82f6b5df28da04f/pyproject.toml#L32-L33

https://peps.python.org/pep-0621/#entry-points
> The `[project.scripts]` table corresponds to the `console_scripts` group in the [entry points specification](https://packaging.python.org/specifications/entry-points/). The key of the table is the name of the entry point and the value is the object reference.